internal/pkg/deploy/cloudformation/lb_fargate_app.go

```diff
@@ -28,25 +49,60 @@ func NewLBFargateStack(in *deploy.CreateLBFargateAppInput) *LBFargateStackConfig
 
 // StackName returns the name of the stack.
 func (c *LBFargateStackConfig) StackName() string {
-	return ""
+	const maxLen = 128
+	n := fmt.Sprintf("%s-%s-%s-app", c.Env.Project, c.Env.Name, c.App.Name)
```

Comment:
Nit: Single-letter variable names give me the heebie-jeebies -- any chance we can call this `stackName` or something more descriptive?

Reply:
Sure, changed to `stackName`